Backpack/Tests/SnapshotTests/BPKShareCardButtonSnapshotTest.swift

```diff
@@ -0,0 +1,95 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+import Backpack
+
+import SnapshotTesting
+
+class BPKShareCardButtonSnapshotTest: XCTestCase {
+    
+    let traitDarkMode = UITraitCollection(userInterfaceStyle: .dark)
```

Comment:
this is not needed right? we can remove the line

Reply:
I removed them in https://github.com/Skyscanner/backpack-ios/commit/6a0bc4e39ae8a1a33f0d5f0247fd4ea4bbea213f